packages/calcite-components/src/components/button/button.scss

```diff
@@ -4,122 +4,51 @@
- * @prop --calcite-button-background-color-hover: Specifies the background color of the component when in hover state.
- * @prop --calcite-button-background-color-focus: Specifies the background color of the component when in focus state.
- * @prop --calcite-button-background-color-active: Specifies the background color of the component when in active state.
  * @prop --calcite-button-text-color: Specifies the text color of the component.
- * @prop --calcite-button-text-color-hover: Specifies the text color of the component when in hover state.
- * @prop --calcite-button-text-color-focus: Specifies the text color of the component when in focus state.
- * @prop --calcite-button-text-color-active: Specifies the text color of the component when in active state.
  * @prop --calcite-button-icon-start-color: Specifies the color of the component's start position icon.
- * @prop --calcite-button-icon-start-color-hover: Specifies the color of the component's start position icon when in hover state.
- * @prop --calcite-button-icon-start-color-focus: Specifies the color of the component's start position icon when in focus state.
- * @prop --calcite-button-icon-start-color-active: Specifies the color of the component's start position icon when in active state.
  * @prop --calcite-button-icon-end-color: Specifies the color of the component's end position icon.
- * @prop --calcite-button-icon-end-color-hover: Specifies the color of the component's end position icon when in hover state.
- * @prop --calcite-button-icon-end-color-focus: Specifies the color of the component's end position icon when in focus state.
- * @prop --calcite-button-icon-end-color-active: Specifies the color of the component's end position icon when in active state.
  * @prop --calcite-button-box-shadow: Specifies the box shadow of the component.
- * @prop --calcite-button-box-shadow-hover: Specifies the box shadow of the component when in hover state.
- * @prop --calcite-button-box-shadow-focus: Specifies the box shadow of the component when in focus state.
- * @prop --calcite-button-box-shadow-active: Specifies the box shadow of the component when in active state.
  * @prop --calcite-button-border-color: Specifies border colors of the component.
- * @prop --calcite-button-border-color-hover: Specifies border colors of the component when in hover state.
- * @prop --calcite-button-border-color-focus: Specifies border colors of the component when in focus state.
- * @prop --calcite-button-border-color-active: Specifies border colors of the component when in active state.
  */
 
 :host {
   --calcite-button-background-color: var(--calcite-color-background);
   --calcite-button-text-color: var(--calcite-color-text-inverse);
   --calcite-button-box-shadow: var(--calcite-shadow-none);
   --calcite-button-border-color: var(--calcite-color-transparent);
-  --calcite-button-corner-radius: var(--calcite-corner-radius-sharp);
+  --calcite-button-corner-radius: var(--calcite-corner-radius);
```

Comment:
Shouldn't we leave this as the `sharp` token? If an app set their `sharp` to be... 4px, it should affect Buttons, Cards, etc.,